tests/infrastructure/instance_types/supported_os/test_rhel_os.py

```diff
@@ -72,6 +72,16 @@ def test_vmi_guest_agent_exists(self, golden_image_rhel_vm_with_instance_type):
         )
 
 
+class TestVmRhelImageVersion:
+    @pytest.mark.dependency(depends=[f"{TESTS_MODULE_IDENTIFIER}::{TEST_START_VM_TEST_NAME}"])
```

Comment:
in another effort pleas start refactoring to drop these dependencies; if the test depends on a running vm create a fixure that will raise if the MV is not running and replace all dependecnies.